content/en/docs/concepts/overview/object-management-kubectl/kustomization.md

```diff
@@ -83,7 +90,7 @@ metadata:
 ```
 
 #### secretGenerator
-Secrets can also be generated from files or literal key-value paris. To generate a Secret from a file, add an entry to `files` list in `secretGenerator`. Here is an example of generating a Secret with a data item from a file.
```

Comment:
nit: line 78 above, small edit:
The generated ConfigMap can be checked by the following command:

Look for other examples of this phrase throughout the doc.

Reply:
updated this sentence in all occurrence